### ISSUE TITLE
ErrorException fwrite(): SSL: Connection reset by peer

### ISSUE THREAD

**Issue by josuesanta08** [developerdynamo/laravel-push-notification#2]
Hi,

When I'm trying to send a message:

$payload->send($tokens);

I'm getting "ErrorException in AbstractClient.php line 120:
fwrite(): SSL: Connection reset by peer". Don't know the reason and if you can help me it will be a lot appreciate it.

Thanks

**Comment by ilvalerione** [comment]
Do you have a valid PEM certificate?

To send ios notification you need a PEM certificate and configure the correct path in config/pushnotification.php

**Comment by josuesanta08** [comment]
Yes, I have a valid certificate located in the storage path, and the correct reference:

"aps" => [
    	/*
    	 * A valid PEM certificate generated from Apple Push Service certificate
    	 */
        "certificate" 	=> storage_path()."/cert.pem",
    		
    	/*
    	 * Password used to generate a certificate
    	 */
        "passPhrase"  	=> "*****"
    ]
